### PR TITLE
Fix workshop organiser formatting + yaml/json paper data

### PR DESCRIPTION
**Pull request by GuyAglionby**
* Fix workshop organisers not showing up on workshop listing page
* Fix formatting of organisers on each workshop's own page (previously was pipe-separated as in the csv files, now comma separated).
*  Fix handling of paper data stored as yaml/json, where authors/keywords/sessions will already be parsed as lists and not require the `.split("|")` that csv does. Markup looks like:

```yaml
- UID: B1xSperKvH
  title: Enabling Deep Spiking Neural Networks with Hybrid Conversion and Spike Timing Dependent Backpropagation
  authors:
  - Author 1
  - Author 2
  abstract: 'lorem ipsum dolor'
  keywords:
  - imagenet
  session:
  - ''
```

```json
[
    {
        "UID": "B1xSperKvH",
        "title": "Enabling Deep Spiking Neural Networks with Hybrid Conversion and Spike Timing Dependent Backpropagation",
        "authors": [
            "Author 1",
            "Author 2"
        ],
        "abstract": "lorem ipsum dolor",
        "keywords": [
            "imagenet"
        ],
        "session": [
            ""
        ]
    }
]
```

### CODE REVIEW

File: main.py [hunk 1]
```diff
@@ -104,27 +104,54 @@ def schedule():
...
 
 
 def format_paper(v):
+    list_keys = ["authors", "keywords", "session"]
+    list_fields = {}
+    for key in list_keys:
+        if isinstance(v.get(key, ""), list):
+            list_fields[key] = v.get(key, "")
+        else:
+            list_fields[key] = v.get(key, "").split("|")
```
Comment (hao-fang) [inline main.py:120]: :nitpick: extract `v.get(key, "")` as a variable

Comment (hao-fang) [reply main.py:120]: oh this is also used in `format_workshop`. Can you make a method so there is less duplicated code?

Comment (GuyAglionby) [reply main.py]: Of course, makes sense!

**Comment by GuyAglionby** [comment]
No hurry, thanks for your comments :)

File: templates/workshop.html [hunk 2]
```diff
@@ -9,7 +9,9 @@ <h2 class="card-title main-title text-center" style="">
       {{workshop.title}}
     </h2>
     <h3 class="card-subtitle mb-2 text-muted text-center">
-      {{workshop.authors}}
+      {% for organizer in workshop.organizers %}
```
Comment (srush) [inline templates/workshop.html:12]: Can you turn this into 

{{ stuffs | join(", ") }}

(apparently that works)

**Pull request merged**